ruby-challenges.rb

```diff
@@ -17,14 +25,39 @@
 # Expected output: 76
 
 nums_array2 = [25, 17, 47, 11]
-# Expected output: 100
+#Expected output: 100
+def add_arr (num_arr)
+  num_arr.map do |value|
```

Comment:
No need for map when using .sum in Ruby.  Also, make sure to print the invocation and not the return value in the method.